### ISSUE TITLE
After upgrade gokapi fails to start

### ISSUE THREAD

**Issue by skibbipl** [Forceu/Gokapi#216]
```
Gokapi v1.9.4 starting
panic: SQL logic error: no such column: ApiKeys.Expiry (1)
```
I upgraded from following docker image:
https://hub.docker.com/layers/f0rc3/gokapi/v1.9.2/images/sha256-5c5287d22600064250677e980c17fa31532990cc02329a840d742694ccb9dd76?context=explore

**Comment by Forceu** [comment]
Thanks for the report. I could reproduce it and it the issue is related to the fix that was applied for #215.

**Comment by Forceu** [comment]
Fixed in 1e43148
